### PR TITLE
Other message links and public feed navbar fix

### PR DESCRIPTION
**Pull request by drewbern**

### CODE REVIEW

File: src/main/webapp/WEB-INF/message-thread.jsp [hunk 1]
```diff
@@ -47,9 +47,9 @@
     <div id="message-container">
       <div class="message-div">
         <div class="message-header">
-          User: <%= parent.getUser() %> --
+          User: <a href="/users/<%= messages.get(i).getUser() %>"><%= messages.get(i).getUser() %></a> --
```
Comment (ArgaAlan) [inline src/main/webapp/WEB-INF/message-thread.jsp]: Why parent.getUser() was changed to messages.get(i).getUser()?

**Review by ArgaAlan — CHANGES_REQUESTED** [review]
Check the parent.getUser change

File: src/main/webapp/WEB-INF/message-thread.jsp (hunk 1 above)
Comment (drewbern) [reply src/main/webapp/WEB-INF/message-thread.jsp]: Oops! Good catch, I'll change that real quick, I was just copy/pasting without thinking

**Review by ArgaAlan — APPROVED** [review]
LGTM

**Comment by drewbern** [comment]
Addresses #111

**Pull request merged**